app/experimenter/templates/experiments/section_normandy.html

```diff
@@ -12,21 +12,23 @@
     </p>
   {% endif %}
 
-  {% if experiment.normandy_id %}
-    <p>
-      <strong class="mr-2">Normandy Recipe ID: </strong>
-      <code>{{ experiment.normandy_id }}</code>
-    <p/>
-    <p>
-      <a href="{{ experiment.delivery_console_recipe_url }}" target="blank" rel="noreferrer noopener">
-        Delivery Console Recipe
-      </a>
-    </p>
+  {% if experiment.normandy_id  %}
+    <p><strong>Normandy Recipe IDs:</strong></p>
 
-    <p>
-      <a href="{{ experiment.normandy_api_recipe_url }}" target="blank" rel="noreferrer noopener">
-        Normandy API Recipe
-      </a>
-    </p>
+      {% for entry in experiment.format_dc_normandy_urls %}
```

Comment:
Ah yeah, this is silently failing on trying to loop over `None` but Django templates try their best to hide errors so it's just being quiet about it.